README.md

```diff
@@ -3,14 +3,34 @@
 ## Objectives
 
 1. Build a functional Rails form using form_tag
+
 2. Pass a route helper as the argument to form_tag
+
 3. Pass an options hash with method to form_tag
+
 4. Use text_field_tag and other form controls to create inputs.
+
 5. Build a new action that renders a form that will submit to create
 
 
-## Notes
+## Instructions
+
+There are two specs for this lab that are currently failing that you need to take from red to green, the specs are located here: ```specs/features/student_spec.rb``` and are in the feature ```form page```.
+
+At a high level you need to build a form to create a new student and have the form redirect back to the new view template and print out the form params to the screen. Below are a few items to keep in mind:
+
+* Make sure to use the ```form_tag``` helper for generating the HTML form with the correct action and HTTP method.
+
+* Draw a ```new``` and ```create``` route for the ```students``` resource, these can be added to the ```resources``` route method.
```

Comment:
For these instructions you can give a lot less. Just explain the URLs that should work, and explain they should be using the form helpers. Also explain which fields the form should have